### ISSUE TITLE
Value of type 'VerticalCardSwiper' has no member 'focussedIndex'

### ISSUE THREAD

**Issue by bugaco** [JoniVR/VerticalCardSwiper#36]
### New Issue Checklist

- [x] I read the [Contribution Guidelines](CONTRIBUTING.md).
- [x] I updated VerticalCardSwiper to the latest version.
- [x] I read the [documentation](https://github.com/JoniVR/VerticalCardSwiper).
- [x] I searched for [existing GitHub issues](https://github.com/JoniVR/VerticalCardSwiper/issues).

### Issue Description

The document shows that:
```
Get the current focussed card index
cardSwiper.focussedIndex
```
But when I use `cardSwiper.focussedIndex`, Xcode shows an error:
```
Value of type 'VerticalCardSwiper' has no member 'focussedIndex'
```
Is there something wrong with me ?
### Environment

- **iOS Version**: 12.1
- **Xcode Version**: 10.14.3
- **VerticalCardSwiper Version**: 0.1.0-beta7

**Comment by JoniVR** [comment]
Correct, I haven't released a new version that includes this, my bad. 
I'll release a new version later today which will include this feature. 🙂

**Comment by JoniVR** [comment]
Hi @zanyzephyr 

Just wanted to let you know I just released version `0.1.0` which should fix this issue for you 🎉 
If you're using Cocoapods, make sure to change the line in your podfile to:
```rb
pod 'VerticalCardSwiper'
```

Let me know if it works now 😄

**Comment by bugaco** [comment]
Hi @JoniVR 
Yeah, I'm using Cocoapods, and I upgraded the version to `0.1.0`.
Then, the `focussedIndex` property works well😃.
